lib/streamlit/config.py

```diff
@@ -531,8 +531,7 @@ def _browser_server_port():
                 which has limitations and is not guaranteed to always work.
                 To get a token for yourself, create an account at
                 https://mapbox.com. It's free! (for moderate usage levels)""",
-    default_val="pk.eyJ1IjoidGhpYWdvdCIsImEiOiJjamh3bm85NnkwMng4M3"
-    "dydnNveWwzeWNzIn0.vCBDzNsEF2uFSFk2AM0WZQ",
+    default_val="",
```

Comment:
Should we say something in the comment about where we get the token? Like "we'll fetch a shared token for you"?

Reply:
Good idea!

Reply:
Actually, after looking at the existing copy: we already tell the user that leaving the value empty will result in us using Streamlit's shared token (`"If you don't do this you'll be using Streamlit's own token..."`). I don't see value in telling the user where that token comes from (fetched vs hardcoded vs whatever else) - it's just an arbitrary implementation detail.